external-dns.yaml

```
@@ -49,6 +55,34 @@ subpackages:
             /opt/bitnami/external-dns/bin/external-dns --version
             /opt/bitnami/external-dns/bin/external-dns --help
 
+  - name: ${{package.name}}-iamguarded-compat
+    pipeline:
+      - uses: iamguarded/build-compat
+        with:
+          package: external-dns
+          version: ${{vars.major-version}}
+      - name: Create symlinks
+        runs: |
+          mkdir -p "${{targets.contextdir}}/opt/iamguarded/external-dns/bin"
+          ln -s /usr/bin/external-dns "${{targets.contextdir}}/opt/iamguarded/external-dns/bin/external-dns"
```

```suggestion
          mkdir -p /opt/iamguarded/external-dns/bin
          ln -s /usr/bin/external-dns /opt/iamguarded/external-dns/bin/external-dns
```
nit and non-blocker: this get moved automatically by finalize-compat pipeline so we can omit explicit contextdir here.